crates/astria-sequencer/src/accounts/state_ext.rs

```diff
@@ -69,8 +74,47 @@ fn nonce_storage_key<T: AddressBytes>(address: T) -> String {
 pub(crate) trait StateReadExt: StateRead + crate::assets::StateReadExt {
     #[instrument(skip_all)]
     async fn get_account_balances(&self, address: Address) -> Result<Vec<AssetBalance>> {
```

Comment:
Not a fan of this code duplication here and with `get_account_balances_ibc_prefixed`. Internally the "natural" representation is in terms of ibc-prefixed items (i.e. `ibc/<sha>`). So the standard way to look at denoms should also be that.

If you want to map the ibc-prefixed items to full trace-prefixed items then that should likely happen at the boundary.